scheduler/src/cook/mesos/api.clj

```diff
@@ -115,11 +117,11 @@
                                     {:resource/type :resource.type/mem
                                      :resource/amount mem}]}]]
     ;; TODO batch these transactions to improve performance
-    @(d/transact conn (-> ports
-                          (into uris)
-                          (into env)
-                          (into maybe-datoms)
-                          (conj txn))))
+    (timers/time! @(d/transact conn (-> ports
```

Comment:
I think you are missing the `submit-jobs-duration` var after timers/time!

Same comment as in submit-to-mesos about counter on jobs and timer on the `doseq`.